lib/seattleflu/id3c/cli/command/clinical.py

```diff
@@ -433,6 +424,210 @@ def add_kp_manifest_data(df: pd.DataFrame, manifest_filenames: tuple, manifest_f
     return df.merge(manifest_data[['barcode', 'enrollid']], how='left')
 
 
+@clinical.command("parse-phskc")
+@click.argument("phskc_filename", metavar = "<PHSKC Clinical Data filename>")
+@click.argument("phskc_specimen_manifest_filename", metavar = "<PHSKC Specimen Manifest filename(s)>")
+@click.argument("geocoding_cache_file", envvar = "GEOCODING_CACHE", metavar = "<Geocoding cache filename>",
+            type = click.Path(dir_okay=False, writable=True))
+
+def parse_phskc(phskc_filename: str, phskc_specimen_manifest_filename: str, geocoding_cache_file: str = None) -> None:
+    """
+    Process clinical data from PHSKC.
+
+    Given a <PHSKC Clinical Data filename> of an Excel document, selects specific
+    columns of interest and reformats the queried data into a stream of JSON
+    documents suitable for the "upload" sibling command.
+
+    All clinical records parsed are output to stdout as newline-delimited JSON
+    records.  You will likely want to redirect stdout to a file.
+    """
```

Comment:
Non-blocker, but would be worthwhile to update the comments here to reflect that parse-phskc command is actually parsing and transforming, removing PII, requires <phskc_specimen_manifest_filename>, and only matching records are returned.  It's quite a bit different than the other parse commands.